pysatModels/__init__.py

```diff
@@ -27,3 +27,8 @@
 # Define a logger object to allow easier log handling
 logging.raiseExceptions = False
 logger = logging.getLogger('pysatModels_logger')
+handler = logging.StreamHandler()
+formatter = logging.Formatter('%(name)s %(levelname)s: %(message)s')
+handler.setFormatter(formatter)
```

Comment:
They could be probably deleted. I added them after looking at https://docs.python.org/3/howto/logging.html and comparing against pysat. Based on experiences that logger sometimes goes wonky.

Reply:
Which option would you prefer?

Reply:
I like the change you made, I was wondering if the `handler` and `format` variables should be deleted at the end of the init file.

Reply:
👍 No worries there. I can uncomment the code (which now removes the vars) and restore the original name. 

My intent was to get at which of the two fixes you prefer. Adding the `handler` and otherwise leave as is, so pysat and pysatModels are doing their own things via logging. Alternately, there is the `pysat.pysatPenumbra` approach, which I forgot to note should allow for control of all pysat packages logging from pysat itself, presuming we spread the technique around.

Reply:
> Alternately, there is the pysat.pysatPenumbra approach, which I forgot to note should allow for control of all pysat packages logging from pysat itself, presuming we spread the technique around.

Conceptually I like this approach better.  I chose this way! 🐴

Reply:
![](https://www.meme-arsenal.com/memes/732d1b1716f4f493280b1ab23b19612d.jpg)